lib/chromiumRebaseL10n.js

```diff
@@ -11,8 +11,12 @@ const resetChromeStringFiles = () => {
   // Revert to original before string replacement for android_chrome_strings.grd
   // because original grd is overwritten with replaced one in ./src/brave.
   const srcDir = config.projects['chrome'].dir
-  const androidChromeStringsPath = path.resolve(path.join(srcDir, 'chrome', 'android', 'java', 'strings', 'android_chrome_strings.grd'))
-  util.run('git', ['checkout', androidChromeStringsPath], { cwd: srcDir })
+  const targetFilesForReset = []
+  targetFilesForReset.push(path.resolve(path.join(srcDir, 'chrome', 'android', 'java', 'strings', 'android_chrome_strings.grd')))
+  targetFilesForReset.push(path.resolve(path.join(srcDir, 'components', 'management_strings.grdp')))
```

Comment:
why do we need to do this?
I have a similar PR replacing `bookmark_bar_strings.grdp` which is also part of `components_strings.grd`
https://github.com/brave/brave-browser/pull/6344

Reply:
cc @petemill

Reply:
Typically, it's a manual step in the rebase process: https://github.com/brave/brave-browser/wiki/Chromium-rebases#fix-strings